_drgn.pyi

```diff
@@ -733,6 +733,41 @@ class Program:
         :param lang: :attr:`Type.language`
         """
         ...
+    def threads(self) -> ThreadIterator:
+        """
+        Return an iterable over all of the threads resident within the process.
+        """
+        ...
+    def thread(self, tid: int) -> Thread:
```

Comment:
As mentioned in the implementation, we want this to accept [integer-like](https://drgn.readthedocs.io/en/latest/api_reference.html#drgn.IntegerLike) objects, too.

```suggestion
    def thread(self, tid: IntegerLike) -> Thread:
```